timescaledb/how-to-guides/continuous-aggregates/adding-automatic-refresh-policies.md

```diff
@@ -9,7 +9,7 @@ use cases. For example, you might want to:
   source data from the hypertable.
 
 These use cases are supported by different configuration to
-[`add_continuous_aggregate_policy`][add-continuous-aggregate-policy].
+[`add_continuous_aggregate_policy`][api-add-continuous-aggregate-policy].
```

Comment:
Shouldn't this have a slash?

```suggestion
[`add_continuous_aggregate_policy`][api/add-continuous-aggregate-policy].
```

Reply:
This reference the links at the bottom of the document so I think the changes are good.